src/Protos/Grpc/redaction.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package event_store.client.redaction;
+option java_package = "com.eventstore.dbclient.proto.redaction";
+
+import "shared.proto";
+
+service Redaction {
```

Comment:
i assume it is intended that the caller will open the switchchunks call to acquire the lock before sending any GetEventPositionReqs, might be worth a quick comment here to explain the intended usage

Reply:
fixed :+1: